define OCAML_OS_TYPE as "xen".  Only a good idea for compiler versions >= 4.04.0.
See https://github.com/mirage/mirage-platform/pull/78/ .
--- config/s.h
+++ config/s.h
@@ -1,4 +1,4 @@
-#define OCAML_OS_TYPE "Unix"
+#define OCAML_OS_TYPE "xen"
 #define OCAML_STDLIB_DIR "/usr/local/lib/ocaml"
 #define POSIX_SIGNALS
 //#define HAS_GETRUSAGE
